cfg/cfg.go

```diff
@@ -82,7 +83,8 @@ func NewFromFile(filePath string) (*Config, error) {
 	}
 
 	// Build logging config
-	loggingConf, err := logging.DefaultConfig()
+	loggingConf := logging.Config{}
+	err = copier.Copy(&loggingConf, &logging.DefaultConfig)
```

Comment:
Why are we using this copier library?

Reply:
We're occasionally modifying the loggingConf variable downstream.  Don't want to accidentally change the default config settings

Reply:
It wouldn't be modified since it's a struct with no pointers: 

```golang
package main

import (
	"fmt"
	"github.com/ava-labs/avalanchego/utils/logging"
)

func main() {
	loggingConf := logging.DefaultConfig
	fmt.Println(loggingConf)
	loggingConf.LogLevel = logging.Info
	fmt.Println(loggingConf)
	loggingConf2 := logging.DefaultConfig
	fmt.Println(loggingConf2)
}
```

```
{24h0m0s 8388608 7 1 false false false false false DEBUG INFO 0 /Users/dhruba/.avalanchego/logs  }
{24h0m0s 8388608 7 1 false false false false false INFO INFO 0 /Users/dhruba/.avalanchego/logs  }
{24h0m0s 8388608 7 1 false false false false false DEBUG INFO 0 /Users/dhruba/.avalanchego/logs  }
```

Reply:
```

var (
	homeDir = os.ExpandEnv("$HOME")
	// DefaultLogDirectory is the default directory where logs are saved
	DefaultLogDirectory = fmt.Sprintf("%s/.%s/logs", homeDir, constants.AppName)

	// DefaultConfig provides a reasonable default logger configuration. It
	// should not be modified, it should be copied if changes are intended on
	// being made.
	DefaultConfig = Config{
		RotationInterval: 24 * time.Hour,
		FileSize:         8 * units.MiB,
		RotationSize:     7,
		FlushSize:        1,
		DisplayLevel:     Info,
		DisplayHighlight: Plain,
		LogLevel:         Debug,
		Directory:        DefaultLogDirectory,
	}
)
```

Gotcha, so safe to ignore this comment on the DefaultConfig declaration?

Reply:
You're already copying it with `:=`

Reply:
Gotcha, do you think there's any concern with them adding a reference type to that struct?

Reply:
No, if they do we can re-assess at that point

Reply:
Alright, it's going to be up to you to catch that ;)